src/sagemaker/serializers.py

```diff
@@ -241,3 +241,27 @@ def serialize(self, data):
         buffer = io.BytesIO()
         scipy.sparse.save_npz(buffer, data)
         return buffer.getvalue()
+
+
+class LibSVMSerializer(BaseSerializer):
+    """Searilize data of various formats to a LibSVM-formatted string."""
+
+    CONTENT_TYPE = "text/libsvm"
+
+    def serialize(self, data):
+        """Serialize data of various formats to a LibSVM-formatted string.
+
```

Comment:
If you decide that adding support for serializing CSV, NumPy, or pandas is unnecessary, it'd be helpful to mention in the docstring that the string or file-like object must already be LibSVM-formatted.